package.json

```diff
@@ -6,7 +6,9 @@
   "bugs": {
     "url": "https://github.com/henrybuilt/react-sticky-table/issues"
   },
-  "dependencies": {},
+  "dependencies": {
+    "styled-components": "^4.3.2"
```

Comment:
what is the difference between emotion and styled-components, or are they the same? My understanding was that they were different, but am seeing a mix here

Reply:
they are both solving the same problem: both are libraries to write CSS in JS. emotions used to be faster, but now seems styled components to be more performant.

I changed it to styled components on this version.